Builtin/LUITabbedFrame.py

```diff
@@ -51,23 +50,41 @@ def add(self, header, frame):
             self.current_frame.show()
         else:
             frame.hide()
+        return header
+
+    def _find_header_index(self, header):
+        # This code is a mess. I blame the absence of StopIteration.
+        # It's also mostly untested.
+        idx = -1
+        cont = True
+        found = False
+        children = self.header_bar.children
+        while cont:
+            idx += 1
+            child = children.next()
+            if child is None:
+                cont = False
+            else:
+                for grandchild in child.children:
+                    if grandchild is None:
+                        break
+                    if grandchild == header:
+                        found = True
+                        cont = False
+                        break
```

Comment:
lui.LUIElementIterator never throws it, but gives back an infinite sequence of Nones instead.